docs/_posts/2019-07-14-Magic-Mirror-Project-Introduction-and-Kick-Off-Notification.md

```diff
@@ -0,0 +1,110 @@
+
+---
+title: Magic Mirror Project Introduction and Kick Off Notification
+date: 2019-07-14 20:00 +0800
+author: newip
+comments: true
+mathjax: true
+categories:
+  - MM Project Kickoff poster
+tags:
+  - MM Project
```

Comment:
Please add a teaser image to this blog post by adding the following front matters:

```yaml
header:
  teaser: /assets/...
```

Reply:
Uploads are disabled.